### ISSUE TITLE
About Novel/bbox_mAP_50

### ISSUE THREAD

**Issue by xiaoyi728** [wusize/ovdet#24]
Hello, when running this project, I set batchsize to [4,4]
As follows:
`batch_split = [4, 4]
      train_dataloader = dict(
      batch_size=sum(batch_split),
      num_workers=sum(batch_split),`
The number of `iterations` is 180000 and the file `schedule_180k.py` is used.
The final `Novel/bbox_mAP_50` has a value of **31**. This is 3 points different from the value given in the paper.
May I ask how I can set bs and the number of iterations to align with the accuracy of the paper?
My device is a single card 3090.

**Comment by wusize** [comment]
For a 180k schedule, we assume the total batch size as [16, 16] and learning rate as 0.02. Adjust the learning rate and batch size following linear rule. However, I guess a single 3090 is not enough for training, where the single-gpu batch size should be [16, 16].
